### ISSUE TITLE
Addonpart issues

### ISSUE THREAD

**Issue by CuriousMike56** [RigsOfRods/rigs-of-rods#3201]
Most addonpart features (conflict checks, tweaking flexbodies/props, moving nodes, and wheel swaps) are no longer functional. Only loading additional meshes works OK. 

Log gives "bad arguments" errors:
```
[RoR|Addonpart] WARNING: file 'Accord_Stanced.addonpart', directive 'addonpart_tweak_node': bad arguments
[RoR|Addonpart] WARNING: file 'BoxDodgeRam_Swamper.addonpart', directive 'addonpart_tweak_wheel': bad arguments
[RoR|Addonpart] WARNING: file 'BoxDodgeRam_Stacks_plain_2_L.addonpart', directive 'addonpart_tweak_flexbody': bad arguments
[RoR|Addonpart] WARNING: file 'BoxDodgeRam_Stacks_plain_2_L.addonpart', directive 'addonpart_unwanted_exhaust': bad arguments
```

Assuming this is caused by the newly introduced bundle syntax (#3171)

**Comment by ohlidalp** [comment]
This is most likely caused by https://github.com/RigsOfRods/rigs-of-rods/commit/cb4f11f29bce0ef7c5012325945018b0bbfb355f - the type checks don't expect this.

**Comment by ohlidalp** [comment]
I tried to reproduce on `master` with Honda Accord + Crazy Ricer Kit. I didn't observe the warning above - that was fixed in 7328ea4afab524a6a05899afc8a3531664c5d51a.

However I did notice a problem with the addonpart file - there's a managed material named "Flares" (without quotes) and since the parser is case-insensitive, it thinks it's actually a `flares` section. This can't be worked around in the parser, managed material names must be put in quotes to avoid clashes going forward.

**Comment by CuriousMike56** [comment]
That `Flares` syntax issue is fixed on the Repository version: https://forum.rigsofrods.org/resources/crazy-ricer-kit.454/

The new meshes and skin appear OK, however the wheel node offset doesn't apply:
![RoR_2024-12-24_20-35-33](https://github.com/user-attachments/assets/10ec1bed-ba44-40ba-a207-8b3fc8d2fb28)
